controllers/ztunnel/ztunnel_controller_test.go

```diff
@@ -337,7 +337,7 @@ func TestDetermineReadyCondition(t *testing.T) {
 	}
 }
 
-func TestApplyImageDigests(t *testing.T) {
+func PTestApplyImageDigests(t *testing.T) {
```

Comment:
nit: can you use t.Skip() and point to https://github.com/istio-ecosystem/sail-operator/issues/581? I think that's clearer